build.gradle.kts

```diff
@@ -74,9 +70,9 @@ tasks.withType<KotlinCompile> {
     kotlinOptions {
         freeCompilerArgs = listOf("-Xjsr305=strict")
         jvmTarget = "17"
-        if (System.getenv("CI") == "true") {
-            kotlinOptions.allWarningsAsErrors = true
-        }
+//        if (System.getenv("CI") == "true") {
```

Comment:
synes disse er greie hvis man bare kjølrer tester lokalt og har noe ubrukt variabel etc

Reply:
De skal inn igjen ja, var for å trigge feil lokalt.